src/components/header/HamburgerNavigation.js

```diff
@@ -0,0 +1,101 @@
+import { Fragment, useState } from 'react';
+import { Link, useNavigate } from 'react-router-dom';
+import './HamburgerNavigation.css';
+import { useAuth, useAuthModal } from '../../hooks';
+
+const HamburgerNavigation = props => {
+  const [isHamburgerNav, setIsHamburgerNav] = useState(false);
+  const { isAuth, logoutHandler } = useAuth();
+  const { showModal } = useAuthModal();
+  
+
+  const backdropClasses = isHamburgerNav
+    ? 'hamburger-menu__backdrop'
+    : 'hamburger-menu__backdrop hidden';
+
+  const hamburgerMenuClasses = isHamburgerNav
+    ? 'hamburger-menu'
+    : 'hamburger-menu hamburger-menu--hide';
+
+  const userLogoutHandler = () => {
+    logoutHandler();
+    setIsHamburgerNav(false);
+  };
+
+  const userLoginHandler = () => {
+    setIsHamburgerNav(false);
+    showModal();
+  };
+
+  return (
+    <Fragment>
```

Comment:
why are you using Fragments when you can use <></> there.

Reply:
Just for readabilitty.